pants-plugins/src/python/internal_backend/sitegen/tasks/sitegen.py

```diff
@@ -234,7 +234,7 @@ def ensure_page_headings_linkable(soup):
 
 def link_pantsrefs(soups, precomputed):
   """Transorm soups: <a pantsref="foo"> becomes <a href="../foo_page.html#foo">"""
-  for (page, soup) in soups.items():
+  for (page, soup) in list(soups.items()):
```

Comment:
should still work without changes, right?

```
$ python3
Python 3.6.1 (v3.6.1:69c0db5050, Mar 21 2017, 01:21:04) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> {1:2,3:4}.items()
dict_items([(1, 2), (3, 4)])
```

Reply:
It should! I missed this and will fix when I get a chance. 

The futurize script is extra conservative because sometimes you do need to wrap in a list, eg when accessing a specific index. Here, the view is preferred and far more performant. 

Thanks for catching this!